src/CHANGELOG

```diff
@@ -8,6 +8,9 @@
 ///    - [yy]: Minor version with non-breaking API and library changes
 ///    - [zz]: Bug fix version with no direct changes to API
 ///
+/// - 2020/04/09 (4.02.1) - Removed unused nk_sqrt function to fix compiler warnings
+/// - 2020/04/09 (4.01.11) - Fixed compiler warnings if you bring your own methods for
+///                        nk_cos/nk_sin/nk_strtod/nk_memset/nk_memcopy/nk_dtoa
```

Comment:
According to the commit changelog, the version `4.01.11` is not separately "checkoutable", so it's really confusing to have it separately in the changelog. Please either make both versions (`4.01.11` `4.02.1`) separately checkoutable or bump the version in this PR just once and squash the version comment messages :wink:.

Reply:
Merged everything into one commit.